### PR TITLE
Feat/downstream controller fix

### PR DESCRIPTION
**Pull request by JiaweiGithub**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

No need for repeated  map operations 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by fisherxu** [comment]
cc @RyanZhaoXB

**Comment by TianTianBigWang** [comment]
/lgtm
please pass the CI.

**Comment by JiaweiGithub** [comment]
> /lgtm please pass the CI.

👌🏻 ci has passed

File: cloud/pkg/devicecontroller/controller/downstream.go [hunk 1]
```diff
@@ -100,14 +100,12 @@ func isDeviceModelUpdated(oldTwin *v1alpha2.DeviceModel, newTwin *v1alpha2.Devic
 // deviceModelUpdated is function to process updated deviceModel
 func (dc *DownstreamController) deviceModelUpdated(deviceModel *v1alpha2.DeviceModel) {
 	value, ok := dc.deviceModelManager.DeviceModel.Load(deviceModel.Name)
-	dc.deviceModelManager.DeviceModel.Store(deviceModel.Name, deviceModel)
+	dc.deviceModelAdded(deviceModel)
```
Comment (chendave) [inline cloud/pkg/devicecontroller/controller/downstream.go:103]: Can you pls move this line up?

**Review by chendave — APPROVED** [review]
/approve
/lgtm
/hold
hold in case you want to address the nit.

**Comment by chendave** [comment]
pls squash the commits.

File: cloud/pkg/devicecontroller/controller/downstream.go (hunk 1 above)
Comment (JiaweiGithub) [reply cloud/pkg/devicecontroller/controller/downstream.go:102]: already update,is that true?

**Comment by JiaweiGithub** [comment]
> pls squash the commits.

done

**Review by chendave — COMMENTED** [review]
/lgtm
/hold cancel
thanks!

**Comment by JiaweiGithub** [comment]
> /lgtm /hold cancel thanks!

Sorry, I submitted an extra blank line, which seems to affect the ci

**Pull request merged**